Sources/PassKit/PassKit.swift

```diff
@@ -330,13 +327,14 @@ public class PassKitCustom<P, D, R: PassKitRegistration, E: PassKitErrorLog> whe
             throw Abort(.badRequest)
         }
         
-        return body.logs
-            .map { E(message: $0).create(on: req.db) }
-            .flatten(on: req.eventLoop)
-            .map { .ok }
+        for log in body.logs {
+            try await E(message: log).create(on: req.db)
+        }
```

Comment:
```suggestion
        try await body.logs.map(E.init(message:)).create(on: req.db)
```
This should be considerably more efficient.